src/modules/UI/selectors.js

```diff
@@ -135,6 +149,10 @@ export const getSceneState = (state: State, sceneKey: string) => {
 }
 
 export const getExchangeRate = (state: State, fromCurrencyCode: string, toCurrencyCode: string): number => {
+  if (fromCurrencyCode === FIO_STR || toCurrencyCode === FIO_STR) {
+    return 1
```

Comment:
Put a comment to fix this. ie 

`// TODO: add real exchange rate`

Reply:
ok